linux_os/guide/system/accounts/accounts-session/accounts_users_home_files_ownership/oval/shared.xml

```diff
@@ -0,0 +1,52 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("All User Files and Directories In The Home Directory Must Have a Valid Owner") }}}
+    <criteria>
+      <criterion test_ref="test_accounts_users_home_files_ownership"
+                 comment="All User Files and Directories In The Home Directory Must Have a Valid Owner"/>
+    </criteria>
+  </definition>
+
+  <unix:password_object id="object_accounts_users_home_files_ownership_objects" version="1">
+    <unix:username datatype="string" operation="not equal">nobody</unix:username>
+    <filter action="include">state_file_ownership_home_directories_interactive_uids</filter>
+  </unix:password_object>
+
+  <unix:password_state id="state_accounts_users_home_files_ownership_interactive_uids" version="1">
+    <unix:user_id datatype="int" operation="greater than or equal">{{{ uid_min }}}</unix:user_id>
```

Comment:
The RHEL7 STIG doesn't explicitly mention `root`: https://vaulted.io/library/disa-stigs-srgs/red_hat_enterprise_linux_7_security_technical_implementation_guide/V-204471?version=V3R4

But shouldn't  the `root` user be checked as well?